meta-lmp-base/recipes-core/images/initramfs-ostree-lmp-recovery.bb

```diff
@@ -0,0 +1,49 @@
+DESCRIPTION = "Linux microPlatform OSTree initramfs recovery image"
+
+inherit core-image nopackages
+
+PACKAGE_INSTALL = " \
+	aktualizr-lite \
+	base-files \
+	base-passwd \
+	busybox \
+	e2fsprogs-e2fsck \
+	e2fsprogs-mke2fs \
+	e2fsprogs-resize2fs \
+	initramfs-framework-base \
+	initramfs-module-ostree-recovery \
+	initramfs-module-udev \
+	ostree \
+	udev \
+	util-linux-mount \
+	${@bb.utils.contains('MACHINE_FEATURES', 'fiovb', 'optee-fiovb', '' , d)} \
+	${@bb.utils.contains('SOTA_CLIENT_FEATURES', 'ubootenv', 'u-boot-fw-utils', '' , d)} \
+	${ROOTFS_BOOTSTRAP_INSTALL}"
+
+
+# Do not pollute the initrd image with rootfs features
+IMAGE_FEATURES = ""
+IMAGE_LINGUAS = ""
+
+export IMAGE_BASENAME = "initramfs-ostree-lmp-recovery"
+
+LICENSE = "MIT"
+
+IMAGE_FSTYPES = "cpio.gz"
+IMAGE_FSTYPES:remove = "wic wic.gz wic.bmap wic.nopt ext4 ext4.gz"
```

Comment:
Yes, see https://github.com/foundriesio/meta-lmp/pull/562.

This could be yet another thing you could have a look once you got time, as it requires changes and upstreaming for some other layers as well.

Reply:
Ok, I will open a ticket for it

Reply:
Sounds good, better to track internally on jira indeed.